docs/source/community/communication.md

```diff
@@ -76,9 +76,8 @@ Our source code is hosted on
 [GitHub](https://github.com/apache/arrow-datafusion). For developers new to
 the project, we have curated a
 [good-first-issue](https://github.com/apache/arrow-datafusion/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
-list to help you get started.
+list to help you get started. You can find datafusion's major designs in docs/rfcs.
```

Comment:
I think the current specifications are actually in https://github.com/apache/arrow-datafusion/tree/master/docs/source/specification

Reply:
Yeah, I move RFCs into /source/specification